Sources/ForageSDK/Foundation/Network/LDManager.swift

```diff
@@ -78,9 +78,9 @@ public class LDManager {
         let vaultPercentage = ld.doubleVariation(forKey: FlagType.vaultPrimaryTrafficPercentage.rawValue, defaultValue: 100.0)
         let randomNum = Double.random(in: 0...100)
         if (randomNum < vaultPercentage) {
-            vaultType = VaultType.btVaultType
-        } else {
             vaultType = VaultType.vgsVaultType
+        } else {
+            vaultType = VaultType.btVaultType
```

Comment:
to confirm, VGS is now the primary?^

Reply:
yes vgs is primary until we start to decide to turn on bt traffic